internal/app/export/export.go

```diff
@@ -78,6 +80,35 @@ func CreateExport(prefix string, runTime time.Time) (Export, error) {
 	return Export{tmpDir: tmpExportDir, fileList: []string{}, runTime: runTime}, err
 }
 
+// a different version to allow exporting to a local folder
```

Comment:
Comment should have the following format 'CreateExportLocal ...' (with an optional leading article)

Reply:
Comment updated with a bit more detail also based on the "CreateExport" comment